packages/fast-colors/src/color-palette.ts

```diff
@@ -29,7 +29,7 @@ export interface ColorPaletteConfig {
 
 export class ColorPalette {
     public static readonly defaultPaletteConfig: ColorPaletteConfig = {
-        baseColor: new ColorRGBA64(0.5, 0.5, 0.5, 1),
+        baseColor: parseColorHexRGB("#808080"),

```

Comment:
Why are you changing this? Seems like extra work to have to parse the string to middle grey.

Reply:
There is something happening regarding rounding that we should probably look into. Color spaces are really strange, but it has to do with the fractions in the luminance calculations or gamma I think. This will actually parse to something like 0.513 (I don’t remember the exact number), but it’s significant that we process all colors the same when it comes to other calculations and placing them relative to another.